src/Validators/CoverageValidator.php

```diff
@@ -112,7 +112,21 @@ function (Validator $context) {
                                     $previousNames = $patient['previous_names'];
                                     $found = false;
                                     foreach ($previousNames as $previousName) {
-                                        if ($previousName['previous_name_first'] == $values['subscriber_fname'] && $previousName['previous_name_last'] == $values['subscriber_lname']) {
+                                        // do a strict equality and then we can do multibyte comparison for localizations
+                                        // note if we want to handle more comprehensive multibytes
+                                        // we need to do some normalizations as per this stackoverflow post: https://stackoverflow.com/a/38855868
+                                        // I'm not incorporating the normalization techniques yet as we need to better understand their implications before
+                                        // applying the fix
+                                        // TODO: look at normalizing the multi-byte strings to better handle international languages where multiple unicode characters correspond to the same uppercase or lowercase character
+                                        if (
+                                            ($previousName['previous_name_first'] == $values['subscriber_fname']
+                                                || mb_strtolower($previousName['previous_name_first']) == mb_strtolower($values['subscriber_fname'])
+                                            )
+                                            &&
+                                            ($previousName['previous_name_last'] == $values['subscriber_lname']
+                                                || mb_strtolower($previousName['previous_name_last']) == mb_strtolower($values['subscriber_lname'])
+                                            )
+                                        ) {
```

Comment:
could  just bite bullet and put following function in sanitize.inc.php (this file always including in all scripts):
```
function mb_is_string_equal_ci($string1, $string2): bool
{
    if ($string1 == $string2) {
        return true;
    }

    $string1_normalized = Normalizer::normalize($string1, Normalizer::FORM_KC);
    $string2_normalized = Normalizer::normalize($string2, Normalizer::FORM_KC);
    return mb_strtolower($string1_normalized) === mb_strtolower($string2_normalized)
            || mb_strtoupper($string1_normalized) === mb_strtoupper($string2_normalized);
}
```

Reply:
then have a nice function to use for when doing name lookups etc. that will support internationalization (and it will get tested over time)

Reply:
could also throw bunch of basic unit testing on function at ci akin to this (nice way to test things without needing to manually run them through the api to test):
https://github.com/openemr/openemr/blob/master/tests/Tests/Unit/PasswordVerifyTest.php

Reply:
I'll look at building out the unit tests and getting this in.

Reply:
@bradymiller I went ahead and wrote a unit test and implemented the mb_is_string_equal_ci function.  Let me know what you think.